### ISSUE TITLE
Error when using templateString without variable

### ISSUE THREAD

**Issue by ngehlert** [fknussel/webpack-version-file#7]
When specifying a template string without a variable an error is thrown

configuration:
```javascript
const version = '1.2.3';
new VersionFile({
     output: './version',
     templateString: `${version}`,
     verbose: true,
 }),
```

```
TypeError: Cannot read property 'map' of null
    at VersionFile.apply (~myDir/node_modules/webpack-version-file/lib/index.js:82:73)
```

I already create a pull request https://github.com/fknussel/webpack-version-file/pull/6

**Comment by ngehlert** [comment]
@fknussel

**Comment by fed** [comment]
Thanks @ngehlert! I've merged your change and published `v0.1.6` 🙂
